homeassistant/components/airthings_ble/coordinator.py

```diff
@@ -58,11 +69,26 @@ async def _async_setup(self) -> None:
             )
         self.ble_device = ble_device
 
+        if "device_model" not in self.config_entry.data:
+            try:
+                _LOGGER.debug("Fetching device info for migration")
+                data = await self.airthings.update_device(self.ble_device)
+            except Exception as err:
```

Comment:
We should be more specific here, ideally we should also update the one in async_update_data but that is outside of the scope